### PR TITLE
Homework 16

### PR DESCRIPTION
**Pull request by DimitryusX**

### CODE REVIEW

File: Homework16/Rectangle.cpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Rectangle::Rectangle(float initialLength, float initialHeight)
+    : m_length(initialLength), m_height(initialHeight) {}
+
+float Rectangle::getArea() {
+    return m_length * m_height;
+}
+
+float Rectangle::getPerimeter() {
+    return 2 * (m_length + m_height);
```
Comment (helen-black) [inline Homework16/Rectangle.cpp:13]: 2.0f

File: Homework16/Team.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#include "Team.h"
+
+Team::Team(const std::string& name, int members) 
+    : m_teamName(name), m_numberOfMembers(members) {};
+
+bool Team::addPlayer(Player* player) {
+    if (m_listOfPlayers.size() < static_cast<size_t>(m_numberOfMembers)) {
+        m_listOfPlayers.push_back(player);
+        return true;
+    }
```
Comment (helen-black) [inline Homework16/Team.cpp:10]: please, check if player isn't already in this team

File: Homework16/Team.cpp [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    else {
+        std::cout << "Team is full." << std::endl;
+        return false;
+    }
```
Comment (helen-black) [inline Homework16/Team.cpp:14]: you don't need `else` here:
```cpp
 std::cout << "Team is full." << std::endl;
        return false;
```

**Review by helen-black — APPROVED** [review]

File: Homework16/Team.cpp (hunk 3 above)
Comment (DimitryusX) [reply Homework16/Team.cpp:14]: @helen-black Fixed.

File: Homework16/Team.cpp (hunk 2 above)
Comment (DimitryusX) [reply Homework16/Team.cpp:10]: @helen-black Fixed.

File: Homework16/Rectangle.cpp (hunk 1 above)
Comment (DimitryusX) [reply Homework16/Rectangle.cpp:13]: @helen-black Fixed.

**Pull request merged**